src/notifications.c

```diff
@@ -17,9 +17,13 @@
 #include "dictionary.h"
 #include "slot_ranges.h"
 #include "asm_state_machine.h"
+//TODO ASM: Review this, maybe is worth duplicating the logic or moving to utils?
+#include "src/coord/rmr/redis_cluster.h"
 
 #define JSON_LEN 5 // length of string "json."
 
+#define TRIMMING_STATE_CHECK_DELAY 100 // 0.1 seconds in milliseconds (We check the trimming state every 0.1 seconds, between MIN_TRIM_DELAY and MAX_TRIM_DELAY)

```

Comment:
Should this be exposed as an internal config? If not, let's validate closesly the perf impact of that